profile_specs/analysis_services_profiles.json

```diff
@@ -1,16 +1,6 @@
 [
   {
-    "name": "Profile_2016_05_16",
-    "resourceTypes": {
-      "Microsoft.AnalysisServices" : {
-        "2016-05-16": ["*"]
-      }
-    },
-    "output_dir": "../../../management/azure_mgmt_analysis_services/lib/profiles",
-    "individual_gem_profile": true
-  },
-  {
-    "name": "Profile_2017_07_14",
+    "name": "V2017_07_14",
```

Comment:
should this name be the date of our release? and maybe prefixed with something different than "V", what if a profile comes out afterwards and the name conflicts with ours? Maybe "Snapshot_2017_10_12", anything shorter would be great, thoughts?

Reply:
I prefer V since it is clean, short and simple. But, I am curious about the name conflict. How could a future version conflict with something of past date????

Reply:
@sarangan12 well, we don't know what date/name could be used for profiles that are not defined by us, for example, we're currently needing to use "v2017_03_09" as version for a "pre-defined" profile, so if sometime later, a profile gets defined with a date that coincides with one of our release dates, it would conflict.